cloud

```diff
@@ -965,21 +987,22 @@ function execute_command() {
             --smtp-password-secret="$smtp_password_secret" \
             --database="$database"
         sections_run "$sections" scheduler_deploy \
-            "$project" "$prefix" "$trigger_topic"
+            "$project" "$prefix" "$load_queue_trigger_topic" "$pick_notifications_trigger_topic"
         sections_run "$sections" submitters_deploy \
             "$project" "$new_topic" "${submitters[@]}"
     elif [ "$command" == "withdraw" ]; then
         sections_run "$sections" submitters_withdraw \
             "$project" "$new_topic" "${submitters[@]}"
-        sections_run "$sections" scheduler_withdraw "$project" "$prefix"
+        sections_run "$sections" scheduler_withdraw "$project" "$prefix" "$pick_notifications_trigger_topic"
```

Comment:
Good job trying to pass `$pick_notifications_trigger_topic` to `scheduler_withdraw`, only it's not actually trying to use it, which we need to fix.